Source/ProjectFirma.Web/Models/GrantAllocationLikelyPerson.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace ProjectFirma.Web.Models
+{
+    public partial class GrantAllocationLikelyPerson : IAuditableEntity
+    {
+        public string AuditDescriptionString => $"Grant Allocation Likely Person: {GrantAllocation?.GrantAllocationID} - {Person?.PersonID}";

```

Comment:
I meant instead of checking the sub-objects that get adding by the code gen for null, just use the object we have the GrantAllocationLikelyPerson object.
so it should be {this.GrantAllocationID} - {this.PersonID}
the code-gen uses those IDs to populate the this.GrantAllocation object, so if the this.GrantAllocationID will always be there and available and you dont need to do null checks